ignite/services/network/publish_test.go

```diff
@@ -183,10 +183,10 @@ func TestPublish(t *testing.T) {
 			Once()
 		suite.CampaignQueryMock.
 			On(
-				"Campaign",
+				"Project",
```

Comment:
This why the tests are failing, the mock call is still `Campaign` (here and 2 other calls below)